unreleased.md

```diff
@@ -2,3 +2,4 @@
 
 - #298 - Fix slider tracker rendering with positive min values
 - #298 - Slider now passes through the attribute 'aria-controls'
+- #294 - Add Cypress tests for the Toggle component
```

Comment:
No need to add this to this document, actually - probably a little confusing in the PR. Since this isn't a change that actually effects the Matchbox API, no need to list it.